tests/testthat/test-qenv_within.R

```diff
@@ -131,3 +131,21 @@ testthat::test_that("within run on qenv.error returns the qenv.error as is", {
 
   testthat::expect_identical(qe, qee)
 })
+
+testthat::describe("within run with `=`", {
+  testthat::it("single expression", {
+    q <- qenv()
+    q <- within(q, {
+      i <- 1
+    })
+  })
+
+  testthat::it("multiple expressions", {
+    q <- qenv()
+    q <- within(q, {
+      j <- 2
+      i <- 1
```

Comment:
@averissimo test claims `multiple expressions`. Do you mean multiple assignemnts, or just more than one assignmnent at atll?
```suggestion
      j <- 2
      i = 1
```

Reply:
Thanks, I meant multiple `=` expressions, corrected in 674d872a